discovery/service/kafka_broker.py

```diff
@@ -106,10 +106,12 @@ def __get_user_dict(self, service_prop: dict, key: str) -> dict:
         # parse line to get the user configs
         for token in jaas_config.split():
             if "=" in token:
-                username, password = token.split('=')
-                if username.startswith('user_'):
-                    principal = username.replace('user_', '')
-                    # Sanitize the password
+                key, value = token.split('=')
+                if key.startswith('username'):
```

Comment:
I can't recall why it was `user_`. Can you please attach some samples in support of this change. I'm not sure if this is non breaking change.

Reply:
thanks. To make it safer, can we keep both `user_` and `username` for now. Do you see any consequences ?

Reply:
yeah, good point. Let me do bit more testing on this.

Reply:
Added back the original code so now it covers both sasl plain form of credentials as well as scram. Tested with both

Reply:
Here's a sample sasl plain jaas config:
listener.name.plain.plain.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="admin" password="admin-secret"  user_admin="admin-secret" user_client="client-secret" user_schema_registry="schema_registry-secret" user_kafka_connect="kafka_connect-secret" user_kafka_rest="kafka_rest-secret" user_ksql="ksql-secret" user_control_center="control_center-secret";